packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -645,4 +658,42 @@ export class ProjectActions {
     debug('navigating to Debug page')
     this.api.routeToDebug()
   }
+
+  /**
```

Comment:
Seems we don't have a centralized place to get info about Cloud projects, so I added this one. Not sure if this is the right pattern - I also think we'd likely want to call this earlier in the code (maybe as soon as we know they are authenticated and they've selected a project?).

Either way, we need to ensure we fetch the cloud project so we can get the name, so I added this function.